d2l-icons.md

```diff
@@ -38,102 +38,103 @@ Size: `18px` x `18`px
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/delete.svg?raw=true) | delete | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/one-drive.svg?raw=true) | one-drive | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/upload.svg?raw=true) | upload |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/discussions.svg?raw=true) | discussions | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/online-rooms.svg?raw=true) | online-rooms | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/user-competencies.svg?raw=true) | user-competencies |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/divider-big.svg?raw=true) | divider-big | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/online.svg?raw=true) | online | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/user-progress.svg?raw=true) | user-progress |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/divider.svg?raw=true) | divider | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/outcomes.svg?raw=true) | outcomes | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/viewed-notviewed.svg?raw=true) | viewed-notviewed |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/dot.svg?raw=true) | dot | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/password.svg?raw=true) | password | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/visibility-hide.svg?raw=true) | visibility-hide |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/download.svg?raw=true) | download | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/pic.svg?raw=true) | pic | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/visibility-show.svg?raw=true) | visibility-show |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/draft.svg?raw=true) | draft | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/pin-filled.svg?raw=true) | pin-filled | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/volume.svg?raw=true) | volume |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/edit.svg?raw=true) | edit | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/pin-hollow.svg?raw=true) | pin-hollow | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/wizard.svg?raw=true) | wizard |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/email-open.svg?raw=true) | email-open | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/preview.svg?raw=true) | preview | | &nbsp; | &nbsp; |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/email.svg?raw=true) | email | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/print.svg?raw=true) | print | | &nbsp; | &nbsp; |
 
 ## tier3
```

Comment:
@dlockhart Hi Dave. I just noticed that `d2l-icons.md` didn't get updated for Tier3 even though I did replace an icon. So now the preview page still shows the old icon. What should I do?

Reply:
Interesting, it should get updated as part of the `npm run build`. Could you try running it again? Not the end of the world if it doesn't get updated, but curious nonetheless.

Reply:
The link https://rawgit.com/BrightspaceUI/icons/master/images/tier3/scorm.svg?raw=true in this file is already pointing to the current icon. But when I inspect the icon used in https://github.com/BrightspaceUI/icons/blob/master/d2l-icons.md, it's pointing to https://camo.githubusercontent.com/2c3ee8597fa2ac1b3f38ecdd71502fe7bcd13fbe/68747470733a2f2f7261776769742e636f6d2f427269676874737061636555492f69636f6e732f6d61737465722f696d616765732f74696572332f73636f726d2e7376673f7261773d74727565, which shows the old icon.

Reply:
Hold up! It just updated. I was just too impatient. It's all good now :)